docs/latest/modules/segmentation.md

```diff
@@ -0,0 +1,3 @@
+## Segmentation {#segmentation}
+
+The segmentation module stores segmentation data in the form of `Labelmap3D` object.
```

Comment:
What's a Labelmap3D?

Reply:
These are the docs I didn't finish yet because you told me to push to `vNext` 😛 .